services/movie.py

```diff
@@ -0,0 +1,42 @@
+from typing import Optional
+
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: Optional = None,
+        actors_ids: Optional = None
+) -> QuerySet:
+    movies = Movie.objects.all()
+    if genres_ids:
+        movies = movies.filter(
+            genres__id__in=genres_ids
+        )
+    if actors_ids:
+        movies = movies.filter(
+            actors__id__in=actors_ids
+        )
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: Optional = None,
```

Comment:
You should declare a type that you expect to obtain, that is wrapped in `Optional`, e.g. `Optional[list[int]]`. Change your type annotations to encounter that.